en/privacy/faq.ftl

```diff
@@ -0,0 +1,72 @@
+privacy-faq-at-mozilla-we-respect-and-protect-desc = At { -brand-name-mozilla } we respect and protect your personal information.
+
+privacy-faq-we-stand-for-people-over-profit = We Stand for People Over Profit.
+privacy-faq-it-can-be-tricky-for-people = It can be tricky for people to know what to expect of any software or services they use today. The technology that powers our lives is complex and people don’t have the time to dig into the details. That is still true for { -brand-name-firefox }, where we find that people have many different ideas of what is happening under the hood in their browser.
+privacy-faq-at-mozilla-we-respect-and-protect = At { -brand-name-mozilla }, we respect and protect your personal information:
+
+# Variables:
+#   $link (url) - link to https://www.mozilla.org/privacy/principles/
+privacy-faq-we-follow-a-set-of-data-privacy = We follow a set of <a href="{ $link }">Data Privacy Principles</a> that shape our approach to privacy in the { -brand-name-firefox } desktop and mobile browsers.
+privacy-faq-we-only-collect-the-data-we = We only collect the data we need to make the best products.
+privacy-faq-we-put-people-in-control-of = We put people in control of their data and online experiences.
+privacy-faq-we-adhere-to-no-surprises-principle = We adhere to “no surprises” principle, meaning we work hard to ensure people’s understanding of { -brand-name-firefox } matches reality.
+privacy-faq-the-following-questions-and = The following questions and answers should help you understand what to expect from { -brand-name-mozilla } and { -brand-name-firefox }:
+privacy-faq-i-use-firefox-for-almost-everything = I use { -brand-name-firefox } for almost everything on the Web. You folks at { -brand-name-mozilla } must know a ton of stuff about me, right?
+privacy-faq-firefox-the-web-browser-that = { -brand-name-firefox }, the web browser that runs on your device or computer, is your gateway to the internet. Your browser will manage a lot of information about the websites you visit, but that information stays on your device. { -brand-name-mozilla }, the company that makes { -brand-name-firefox }, doesn’t collect it (unless you ask us to).
+privacy-faq-really-you-dont-collect-my-browsing = Really, you don’t collect my browsing history?
+
+# Variables:
+#   $link (url) - link to https://addons.mozilla.org/firefox/addon/firefox-pioneer/
+privacy-faq-mozilla-doesnt-know-as-much = { -brand-name-mozilla } doesn’t know as much as you’d expect about how people browse the web. As a browser maker, that’s actually a big challenge for us. That is why we’ve built opt-in tools, such as <a href="{ $link }">{ -brand-name-firefox } Pioneer</a>, which allows interested users to give us insight into their web browsing. If you sync your browsing history across { -brand-name-firefox } installations, we don’t know what that history is - because it’s encrypted by your device.
```

Comment:
For the very last part of this string, should it be an `—` (emdash), not an `-` (endash)?